keps/prod-readiness/sig-storage/284.yaml

```diff
@@ -0,0 +1,7 @@
+kep-number: 284
+alpha:
```

Comment:
Did we have PRR for alpha and beta? If not, I think we can remove those entries.

Reply:
removed.